kubernetes/start

```diff
@@ -71,5 +76,23 @@ fi
 sudo mkdir -p $PWD/tmp/mariadb-socket
 sudo chmod 777 $PWD/tmp/mariadb-socket
 
+if docker images | grep "^sles12/velum" &> /dev/null ; then
+  log "Velum development image is present"
+else
+  log "Building Velum development image..."
+  CONTAINER_BUILD=`docker run -v $(readlink -f $PWD/..):/srv/velum -d docker-testing-registry.suse.de/sles12/velum:development \
+                    bash -c "mkdir -p /var/lib/velum/ruby/2.1.0 && gem install --no-ri --no-rdoc --install-dir /var/lib/velum/ruby/2.1.0 bundler && \
+                    ln -sf /var/lib/velum/ruby/2.1.0/bin/bundler.ruby2.1 /bin/bundle && \
+                    bundle config --local frozen 0 && bundle config --local build.nokogiri --use-system-libraries && \
+                    bundle install --binstubs=/usr/local/bin --deployment --path=/var/lib/velum && \
+                    wget https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 -P /opt && \
+                    tar -xjf /opt/phantomjs-2.1.1-linux-x86_64.tar.bz2 -C /opt && \
+                    mv /opt/phantomjs-2.1.1-linux-x86_64/bin/phantomjs /usr/local/bin && \
+                    rm -rf /opt/phantomjs-2.1.1-linux-x86_64"`
+  docker logs -f $CONTAINER_BUILD
```

Comment:
Yes, once the build has finished, the logs follow will end (that way we also wait for the build to finish in order to commit it)